pkg/tf/serialization/tf_resource_test.go

```diff
@@ -34,46 +31,21 @@ func TestSerializeCluster(t *testing.T) {
 	testSerialize(t, "testdata/cluster.terraform.instancestate", "google_container_cluster", "testdata/cluster.golden.tf")
 }
 
-func testSerialize(t *testing.T, instanceStateFile, tfType, goldenFile string) string {
+func testSerialize(t *testing.T, instanceStateFile, tfType, goldenFile string) {
 	provider := tfprovider.NewOrLogFatal(tfprovider.UnitTestConfig())
-	b, err := ioutil.ReadFile(instanceStateFile)
-	if err != nil {
-		t.Fatalf("failed to load instance state for instance, %s", err.Error())
-	}
+	b := test.MustReadFile(t, instanceStateFile)
```

Comment:
I'm unsure if turn error handling to an util function(MustReadFile) is best practice. My thought is handling errors after calling ReadFile is more straightforward. This also allows us to customize the error message to include more details, i.e. specific files being read, which is useful for debugging purposes.

Reply:
This only works in tests because of t.Fatalf being allowed in tests.  I don't know what we would print that isn't "error reading file %q: %v" though?  The contents of the file don't really matter IMO